testsuite/MDAnalysisTests/lib/test_nsgrid.py

```diff
@@ -371,3 +376,35 @@ def test_issue_2670():
 
     # should return the one atom overlap
     assert len(ag2.select_atoms('around 0.0 resid 3')) == 1
+
+
+def high_mem_tests_enabled():
+    """ Returns true if ENABLE_HIGH_MEM_UNIT_TESTS is set to true."""
+    env = os.getenv("ENABLE_HIGH_MEM_UNIT_TESTS", default="0")
+    try:
+        return strtobool(env)
+    except ValueError:
+        return False
+
+
+reason = ("Turned off by default. The test can be enabled by setting the ENABLE_HIGH_MEM_UNIT_TESTS "
```

Comment:
We can ignore line 73, but this is a pretty big PEP8 characters per line violation, can you split this over 3 lines instead please?